index.html

```diff
@@ -9142,6 +9161,252 @@
                 </pre>
     </section>
   </section>
+  <section id="quic-stream*">
+    <h2><dfn>RTCQuicStream</dfn> Interface</h2>
+    <p>The <code>RTCQuicStream</code> includes information relating
+    to QUIC stream.</p>
+    <section id="rtcquicstream-overview*">
+      <h3>Overview</h3>
+      <p>An <code><a>RTCQuicStream</a></code> instance is associated to
+      an <code><a>RTCQuicTransport</a></code> instance.</p>
+    </section>
+    <section id="rtcquicstream-operation*">
+      <h3>Operation</h3>
+      An <code><a>RTCQuicStream</a></code> instance is created by the
+      <code><a>RTCQuicTransport</a></code> with a QUIC stream underneath.
+      When writing, the <code><a>RTCQuicStream</a></code>  will buffer
+      the data if the transport underneath is not writable. When receiving,
```

Comment:
Should probably note that waitForWritable must be called before write() or write() will (likely) fail.